pages/common/banner.md

```diff
@@ -0,0 +1,15 @@
+# banner
+
+> Print the given argument as a large ASCII art.
+
+- Print the text message as a large banner. Note: quotes are optional:
+
+`banner {{"Hello World"}}`
+
+- Print the text message as a banner with a width of 50 characters:
+
+`banner -w 50 {{"Hello World"}}`
+
+- Read text from stdin and print it as a large banner:
```

Comment:
Just "Read text from stdin" is fine

Reply:
Thanks for the suggestion, but - just for my understanding - you mean that "and print it as a large banner" here is redundant because there are other examples before?

Also, I wonder if there is a general rule for it - see, for example, the pages of commands _gist_ and _bash_. The review process would be faster for you if there is a shared, consistent way to describe examples and descriptions. What do you think?

Reply:
Your understanding is correct.

In general, if we have explained something 2 times, there is no need to repeat that as it becomes implicit. However, it is a rule of thumb and depends on the context and the page. Therefore, we don't have it as a written rule.

I apologize if you have looked through other pages and seen inconsistencies. Sometimes, other pages have been committed years ago and reviewers also miss something. We always try to keep everything consistent, but sometimes things slip through cracks.

Thanks for your understanding. And I appreciate you taking the time to look through other pages !